doc/users/install.rst

```diff
@@ -119,7 +125,7 @@ Must Have
 Nibabel_ 1.0 - 1.4
```

Comment:
should probably allow for greater nibabel version. some of these are set with `*_MIN_VERSION` in info.py. we should just follow that i think.